zerver/openapi/zulip.yaml

```diff
@@ -6879,13 +6879,14 @@ paths:
             organization. Zulip Cloud users can request the `can_create_users`
             permission for a bot by contacting [Zulip Cloud support][support]
             with an explanation for why it is needed. Self-hosted installations
-            can toggle `can_create_users` on an account using the `manage.py
-            change_user_role` command.
+            can toggle `can_create_users` on an account using the [`manage.py
+            change_user_role` command][management-commands].
```

Comment:
I think I'd prefer for the link to be just on the text "management command", i.e.  "using the `manage.py change_user_role` [management command](...)." -- that feels to me like a more readable pattern.